src/SdkConfig.ts

```diff
@@ -30,6 +30,7 @@ export const DEFAULTS: IConfigOptions = {
     jitsi: {
         preferred_domain: "meet.element.io",
     },
+    element_call_url: "https://call.element.io",
```

Comment:
Do we want to group all future EC settings or do we want to keep them flat?
If we want to group them we should probably introduce a dedicated level now:

```suggestion
    element_call: {
        url: "https://call.element.io",
    },
```

Reply:
Great point, there's at least 1 other Element Call related setting that we'll need to add, so that makes sense